tests/testcases/040_check-network-adv.yml

```diff
@@ -70,7 +70,8 @@
       when:
         - agents.content != '{}'
 
-    - debug: var=result.content|from_json
+    - debug:
```

Comment:
Unfortunately I think this change is masking errors since the new output no longer matches the `failed_when` condition.

```
TASK [Check netchecker status] *************************************************
task path: /kargo-ci/kubernetes-sigs-kubespray/tests/testcases/040_check-network-adv.yml:61
Saturday 12 January 2019  16:02:17 +0000 (0:00:00.096)       0:00:02.386 ****** 
ok: [instance-3 -> None] => {"censored": "the output has been hidden due to the fact that 'no_log: true' was specified for this result", "changed": false}

TASK [debug] *******************************************************************
task path: /kargo-ci/kubernetes-sigs-kubespray/tests/testcases/040_check-network-adv.yml:73
Saturday 12 January 2019  16:02:17 +0000 (0:00:00.448)       0:00:02.835 ****** 
ok: [instance-3 -> None] => {
    "msg": "Error occured while checking the agents. Details: Get https://10.233.0.1:443/api/v1/pods?labelSelector=app+in+%28netchecker-agent%2Cnetchecker-agent-hostnet%29: dial tcp 10.233.0.1:443: getsockopt: connection refused\n"
}
```

While this is better than failing due to a bug in `from_json`, the task should still fail when it encounters that error.

Reply:
Any recommendations ?

Reply:
maybe add some debug statements to pull the flannel, kube-proxy, and netchecker server logs